packages/e2e-tests/puppeteer/__tests__/rps.test.ts

```diff
@@ -2,7 +2,7 @@
 import {Page, Browser} from 'puppeteer';
 import {configureEnvVariables, getEnvBool} from '@statechannels/devtools';
 
-import {setUpBrowser, loadRPSApp} from '../helpers';
+import {setUpBrowser, loadWeb3App} from '../helpers';
```

Comment:
Not sure you want to touch this file?

Reply:
It's a simple rename so I can reuse the function.

Reply:
Ah I see. You're using `Web3App` to stand for rps or web3torrent or something of that type. Other possible names:

- `loadApp` 
- `loadDapp`
- `loadStateChannelApp`.